### PR TITLE
Editorial suggestions (IAB review)

### PR DESCRIPTION
**Pull request by tfpauly**
These are editorial suggestions for typos / clarity from my review of the document.

### CODE REVIEW

File: draft-iab-protocol-maintenance.md [hunk 1]
```diff
@@ -95,13 +95,22 @@ robustness principle to avoid interoperability issues.
...
-serves primarily as a record of the hazards inherent in applying the robustness
-principle and to offer an alternative strategy for handling interoperability
+serves primarily as a record of the hazards in applying the robustness principle
+too broadly, and offers an alternative strategy for handling interoperability
 problems in deployments.
 
-Ideally, protocol implementations never have to apply the robustness principle.
-Or, where it is unavoidable, use of the robustness principle is viewed as a
-short term workaround that needs to be quickly reverted.
+Ideally, protocol implementations that can be actively maintained never have
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:102]: This is a pretty big change in stance.  Perhaps that was deliberate, but I prefer this framing.

```suggestion
Ideally, protocol implementations can be actively maintained and never have
```

File: draft-iab-protocol-maintenance.md [hunk 2]
```diff
@@ -95,13 +95,22 @@ robustness principle to avoid interoperability issues.
...
+to apply the robustness principle.  The robustness principle may still need
+to used as a short-term mitigation for deployments that cannot yet be easily
+updated and do not yet have documented specifications for workarounds, but
+such cases need not be permanent. This is discussed further in {{active}}.
+
+It is important to note that not applying the robustness principle does
+not mean that implementations will be inflexible to new input. As discussed
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:109]: Double-negation.
```suggestion
Avoiding use of the robustness principle does
not mean that implementations will be inflexible to new input. As discussed
```

**Review by martinthomson — COMMENTED** [review]
This is very good.  Thanks go to everyone who contributed.

One of the changes was a little too much of a tweak for my comfort, but I'm open to hearing why what you proposed was better.

File: draft-iab-protocol-maintenance.md (hunk 1 above)
Comment (tfpauly) [reply draft-iab-protocol-maintenance.md:102]: I’m fine with that tweak—mainly I think it’s important to mention the active maintenance here, and having it be an “and” still achieves that while maintaining the strength of the statement.

File: draft-iab-protocol-maintenance.md [hunk 3]
```diff
@@ -120,47 +129,49 @@ This formulation of the principle expressly recognizes the possibility that the
...
 # Protocol Decay
 
-The application of the robustness principle to the early Internet, or any system
-that is in early phases of deployment, is expedient.  Applying the principle
-defers the effort of dealing with interoperability problems, which prioritizes
-progress.  However, deferral can amplify the ultimate cost of handling
-interoperability problems.
+The application of the robustness principle to any system that is in early
+phases of deployment, such as the early Internet, is expedient.  Applying the
+principle defers the effort of dealing with interoperability problems and
```
Comment (martinthomson) [inline draft-iab-protocol-maintenance.md:154]:
```suggestion
principle defers the effort of dealing with interoperability problems and
```

File: draft-iab-protocol-maintenance.md [hunk 4]
```diff
@@ -80,12 +80,12 @@ design of interoperable protocols in general.
 Time and experience shows that negative consequences to interoperability
 accumulate over time if implementations apply the robustness principle.  This
 problem originates from an assumption implicit in the principle that it is not
-possible to affect change in a system the size of the Internet.  That is, the
-idea that once a protocol specification is published, changes that might require
-existing implementations to change are not feasible.
+possible to effect change in a system the size of the Internet.  It assumes
```
Comment (DavidSchinazi) [inline draft-iab-protocol-maintenance.md:83]: Very minor nit: while effect is also correct, I find affect to be clearer. Happy to be overruled by native speakers (in which case I'll learn something :-) ) and happy to have this merged with either effect or affect

```suggestion
possible to affect change in a system the size of the Internet.  It assumes
```

**Review by DavidSchinazi — APPROVED** [review]
Thank you Tommy, this PR is really great and I think it addresses the concerns raised during the IAB retreat without significantly watering down the message. Well done!

File: draft-iab-protocol-maintenance.md (hunk 4 above)
Comment (tfpauly) [reply draft-iab-protocol-maintenance.md:83]: This is definitely "effect change", IMO =)

https://writingexplained.org/affect-change-or-effect-change

> What does affect change mean? Affect change is an incorrect version of the phrase "effect change".

Comment (DavidSchinazi) [reply draft-iab-protocol-maintenance.md:83]: Ah I see. `affect a system` means roughly the same as `effect change in a system` whereas `affect change` is redundant. Thank you :) Marking as resolved.

**Comment by DavidSchinazi** [comment]
@martinthomson feel free to merge this and submit a revision if you think this PR looks good

**Pull request merged**